### ISSUE TITLE
German locale contains messages in Dutch

### ISSUE THREAD

**Issue by Brow71189** [justreportit/thunderbird#7]
This makes it very hard to use the plug-in on a PC set to German.

**Comment by Bernd95** [comment]
The file message.json in the directory de should be moved to a new directory nl. Instead use the attached file 
[messages.json.de.zip](https://github.com/justreportit/thunderbird/files/5977590/messages.json.de.zip) in the directory de. It is a first try to translate everything to German. I must admit that sometimes I did not understand the meaning of certain sentences or terms – therefore I left some words in English. Feel free to either improve this translation or explain the missing terms. In particular:
-	What is a custom lookup server and what is it looking up?
-	What is the Whois-Option for?
Also: It seems that the options for the addresses in the first tab (Registrar, Spam Cop,…) are not marked as “to be translated”.

**Comment by nicprov** [comment]
It is definitely possible that I mixed up the languages, I must admit, I did rush the translation part a little!
@Bernd95 Thanks a lot for the work! I'll definitely add it as part of the next release. And to answer your questions:
- _What is a custom lookup server and what is it looking up?_ -> The custom lookup server performs a lookup in the WHOIS database (a registry for domains) and attempts to find the domain registrars abuse email to report the spam to. This method often results in the spam domain being banned permanently (effectively tackling spam at the source)
- _What is the Whois-Option for?_ -> This tab is used to specify which whois lookup website to use in the event of a failed lookup with the custom lookup server. Of course, this is only shown if this option is enabled
- _Also: It seems that the options for the addresses in the first tab (Registrar, Spam Cop,…) are not marked as “to be translated”._ -> You are right, I was worried that the translation would make it more difficult to figure out what the options are, but I can definitely add these items as part of the language pack.
